docs/io/ascii/howtoexamples.rst

```diff
@@ -0,0 +1,209 @@
+.. include:: references.txt
+
+.. _io_ascii_how_to_examples:
+
+How to fix a table that is not read or written correctly
+********************************************************
+
+The purpose of this section is to provide a few examples how we can
+deal with tables that fail to read or cannot be written with the
+default settings.
+
+Obtain the data table in a different format
```

Comment:
When I encounter a table that does not read, trying `guess=False` is usually not the first thing I do. I usually try to get a different format (fits, csv instead of space-separated etc.) first because then I don't have ot find out what the options are that allow me to read it with the ascii readers.